ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -181,14 +185,31 @@ const DATASET_TYPE_PANEL_PROPS = {
       fields: SNP_QUALITY_FILTER_FIELDS,
     },
   },
+  [DATASET_TYPE_VARIANT_MITO]: {
```

Comment:
I think all the other panels for `DATASET_TYPE_VARIANT_MITO` need to have the same panels as `DATASET_TYPE_VARIANT_CALLS`. I think the cleanest way to do that is to do the assignment for this panel type after the dict has been created:
```
const DATASET_TYPE_PANEL_PROPS = {
  [DATASET_TYPE_VARIANT_CALLS]: {...},
  [DATASET_TYPE_VARIANT_SV]: {...},
}
DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_MITO] = {
  ...DATASET_TYPE_PANEL_PROPS[DATASET_TYPE_VARIANT_CALLS],
  FREQUENCY_PANEL.name]: {
  ...
}
```

Reply:
Do we have an `SV` only case? If yes, I have to update a little bit to my last commit.

Reply:
no, we do not have an only SV or an only MITO case, but thanks for checking